addons/website_event/views/event_templates_list.xml

```diff
@@ -97,13 +97,11 @@
                                     </button>
                                 </h2>
                                 <div t-attf-class="o_wevent_offcanvas_cat_#{category.name} accordion-collapse collapse">
-                                    <div class="accordion-body pt-0">
+                                    <div class="accordion-body bg-white pt-0">
                                         <ul class="list-group list-group-flush">
                                             <li t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids)" class="list-group-item">
                                                 <t t-foreach="category.tag_ids" t-as="tag">
-                                                    <a t-if="tag.color"
```

Comment:
Reminder: Like in the prev. PR, this code will output:
```xml
<ul>
     <li>
         <a/>
         <a/>
         <a/>
         [...]
    </li>       
 </ul>
 ```
 Let's not forget it for the followup task on `master` 😉